### PR TITLE
Unit test for guard.js.

### LINKED ISSUE #631: Add Unit Tests: guard.js
We need unit tests for https://github.com/DemocracyLab/CivicTechExchange/blob/master/common/components/utils/guard.js

### PR DESCRIPTION
**Pull request by zh-zhang916**
As the title says.

close #631

### CODE REVIEW

File: common/components/test/utils.test.js [hunk 1]
```diff
@@ -196,4 +198,11 @@ describe("utils", () => {
     word = utils.pluralize("apple", "apples", 5);
     expect(word).toEqual("apples");
   }); 
+
+  test("guard.duplicateInput", () => {
+    _.debounce = jest.fn();
+    const testFunc = () => "test";
+    Guard.duplicateInput(testFunc);
+    expect(_.debounce).toBeCalledWith(testFunc, 1000, {leading: true});
```
Comment (marlonkeating) [inline common/components/test/utils.test.js:206]: I would instead test that testFunc was called only once after multiple invocations of Guard.duplicateInput's return function.

**Review by marlonkeating — CHANGES_REQUESTED** [review]
The current test is only testing the implementation details, where we want to test the behavior regardless if the implementation changes in the future.

**Review by marlonkeating — CHANGES_REQUESTED** [review]
Test looks good so far.  There's a couple more things we should test

## Checklist ##
- [x] Verify return value of function is duplicated
- [ ] Verify function is called again after timeout period elapses

**Comment by zh-zhang916** [comment]
Fixed.

**Review by marlonkeating — APPROVED** [review]

**Pull request merged**